utils/Command/MissingInSetCommand.php

```diff
@@ -63,7 +63,6 @@ final class MissingInSetCommand extends Command
         __DIR__ . '/../../vendor/rector/rector-phpunit/rules/CodeQuality' => __DIR__ . '/../../vendor/rector/rector-phpunit/config/sets/phpunit-code-quality.php',
 
         // rector-downgrade
-        __DIR__ . '/../../vendor/rector/rector-downgrade-php/rules/DowngradePhp71' => __DIR__ . '/../../vendor/rector/rector-downgrade-php/config/set/downgrade-php71.php',
```

Comment:
`config/set/downgrade-php71.php` no longer exists, so it must be unregistered from `MissingInSetCommand`